src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -151,7 +153,8 @@ private void setupPanel(JabRefFrame frame, BasePanel bPanel, boolean addKeyField
                         false);
                 defaultHeight = 0;
             } else {
-                fieldEditor = new TextArea(field, null);
+                String prompt = (field.equals(FieldName.AUTHOR)) ? Localization.lang("Firstname Lastname and Firstname and Lastname and others") : "";
```

Comment:
Wrong text

It should be `Firstname Lastname and Firstname Lastname`. ` and others` must not be translated as it is a fixed bibtex term used to write `et al.` or something else as configured in the bibtex style.

The rendering of your example is
Lastname, F.; Firstname; Lastname & others
but it should be
Lastname, F.; Lastname, F. & others